packages/bun-usockets/src/internal/loop_data.h

```diff
@@ -18,6 +18,9 @@
 #ifndef LOOP_DATA_H
 #define LOOP_DATA_H
 
+#include <pthread.h>
```

Comment:
if we use atomics we can avoid the mutex and this include (which is probably causing the windows build failure)